File: Sandbox.Modules.CSharp/General/HelloModule.cs
using Sandbox.Core;

namespace Sandbox.Modules.CSharp.General {
    [SandboxModule("Hello World", "cs-hello", "A simple module to test using reflection to discover modules.")]
    [ModuleTags("general", "sandbox", "testing")]
    public class HelloModule : SandboxModule {
        protected override void Execute() {
            SendResponse(SandboxEventType.None, "Hello World!");
        }
    }
}

File: Sandbox.Modules.VisualBasic/General/HelloModule.vb
Imports Sandbox.Core
Namespace Sandbox.Modules.VisualBasic.General
    <SandboxModule("Hello World", "vb-hello", "This module is a simple test of integrating the Visual Basic module collection with the module service.")>
    <ModuleTags("general", "sandbox", "testing")>
    Public Class HelloModule
        Inherits SandboxModule

        Protected Overrides Sub Execute()
            SendResponse(SandboxEventType.None, "Hello from Visual Basic!")
        End Sub
    End Class
End Namespace